builder/package.json

```diff
@@ -3,14 +3,19 @@
     "version": "1.0.0",
     "description": "",
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.2",
         "bootstrap": "^4.3.1",
+        "github-markdown-css": "^4.0.0",
         "gulp": "^4.0.2",
         "gulp-clean-css": "^4.2.0",
+        "gulp-concat": "^2.6.1",
+        "gulp-order": "^1.2.0",
         "gulp-sass": "^4.0.2",
-        "node-sass": "^4.12.0",
         "gulp-uglify": "^3.0.2",
-        "gulp-concat": "^2.6.1",
-        "gulp-order": "^1.2.0"
+        "jquery": "^3.5.1",
+        "node-sass": "^4.12.0",
+        "popper.js": "^1.16.1",
```

Comment:
I'm not sure if popper.js is required, Bootstrap includes a [list of components](https://getbootstrap.com/docs/4.3/getting-started/introduction/#js) that require it